eclipse/openvsx
# Mirror functionality

Fixes #505

### Testing steps
- Change the `ovsx.data.mirror.schedule.extensions` and `ovsx.data.mirror.schedule.metadata` cron expressions. Current time + 3 minutes works well for immediate testing.
- Let the server run, the mirror jobs contain log statements to show what is happening in the background.
- View a fully mirrored extension in the webui. The data should be the same (download count, reviews, rating, verified extension versions, published date, latest update date) even though the layout is a little different.
- View a fully mirrored extension in VS Code. Again the data should be the same, enhanced even because of the newer features.
- Restart the server with a different schedule.

It's also a good idea to run mirror mode for a longer time on `staging` to see what pops up.
